### PR TITLE
Add a very basic admin for abuse reports

### LINKED ISSUE #5601: Add basic admin for abuse reports
QA asked for a way to delete abuse reports in #5473 ; it might prove useful for admins too in the future, to delete spam.

### PR DESCRIPTION
**Pull request by diox**
Fix #5601 

abuse_list
abuse_detail

### CODE REVIEW

File: src/olympia/abuse/admin.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class AbuseReportAdmin(admin.ModelAdmin):
+    raw_id_fields = ('addon', 'user', 'reporter')
+    readonly_fields = ('ip_address', 'message', 'created', 'addon', 'user',
+                       'reporter')
+    list_display = ('reporter', 'ip_address', 'type', 'target', 'message',
+                    'created')
+    actions = ('delete_selected',)
+
```
Comment (eviljeff) [inline src/olympia/abuse/admin.py]: Suggestion : have reporter say 'anonymous user' or something when it's not set.

File: src/olympia/abuse/admin.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from django.contrib import admin
+from .models import AbuseReport
+
+
+class AbuseReportAdmin(admin.ModelAdmin):
+    raw_id_fields = ('addon', 'user', 'reporter')
+    readonly_fields = ('ip_address', 'message', 'created', 'addon', 'user',
+                       'reporter')
+    list_display = ('reporter', 'ip_address', 'type', 'target', 'message',
+                    'created')
```
Comment (eviljeff) [inline src/olympia/abuse/admin.py]: suggestion: add `list_filter = ('type',)`?

Comment (diox) [inline src/olympia/abuse/admin.py]: Good idea. https://github.com/mozilla/addons-server/pull/5602/commits/563307b0afe2dd79dad43d14a778de5af2e81c55

File: src/olympia/abuse/admin.py (hunk 1 above)
Comment (diox) [inline src/olympia/abuse/admin.py]: It currently says "(None)" which I thought was good enough.

**Comment by diox** [comment]
It's certainly more of a quick hack rather than a full product, but @wagnerand seemed to think it would help triaging incoming abuse reports compared to now where admins have to do it by email and can't do mass-deletion of spam for instance.

**Pull request merged**